config/express.js

```diff
@@ -21,7 +21,64 @@ module.exports = (app, config) => {
     watch: true,
   });
 
-  app.use(helmet());
+  app.use(helmet.contentSecurityPolicy({
+    directives: {
+      defaultSrc: [
+        '\'self\'',
+      ],
+      scriptSrc: [
+        '\'self\'',
+        'data:',
+        'www.google-analytics.com',
+        's.webtrends.com',
+        'statse.webtrendslive.com',
+        'static.hotjar.com',
+        'script.hotjar.com',
+        'cdn.jsdelivr.net',
+      ],
+      imgSrc: [
+        '\'self\'',
+        'data:',
+        'www.google-analytics.com',
+        'statse.webtrendslive.com',
+        'hm.webtrends.com',
+      ],
+      styleSrc: [
+        '\'self\'',
+        'fast.fonts.net',
+        'https://dhrlmnmyf2njb.cloudfront.net/',
```

Comment:
Note: This URL is hard coded in the CSS too. Have raised an issue to make it configurable through an env var.